yt/utilities/amr_kdtree/amr_kdtree.py

```diff
@@ -128,7 +128,7 @@ def check_tree(self):
             nre = self.ds.arr(node.get_right_edge(), units="code_length")
             li = np.rint((nle - gle) / dds).astype("int32")
```

Comment:
outside of scope for this PR but my eyebrows raised a little seeing this casting to int32 instead of int64.

Reply:
yeah, fortunately these variables don't seem to be actually used (except for that one `assert np.all(dims > 0)` which is runtime-skippable)